extension/war/src/main/webapp/groovy/social/webui/activity/UIActivityCommentActions.gtmpl

```diff
@@ -69,8 +69,8 @@
         </li>
         <% _ctx.includeTemplates("UIActivityCommentActionBar-actions-after") %>
         <li class="dateTime">
-            <a id="EditCommentTime${comment.id}" rel="tooltip" data-placement="bottom" title="<%=if(comment.postedTime != comment.updated.getTime() ){ _ctx.appRes("UIActivity.label.PostedFrom").replace("{0}",commentAbsolutePostedTime) } else { commentAbsolutePostedTime }%>" href="<%=uicomponent.getActivityPermalink(activityID)%>" <% if(comment.postedTime != comment.updated.getTime() ){%>> $commentEditedTime <%} else {%> > ${commentPostedTime} <%}%></a>
+            <a class="editCommentTimeComment" id="EditCommentTime${comment.id}" rel="tooltip" data-placement="bottom" title="<%=if(comment.postedTime != comment.updated.getTime() ){ _ctx.appRes("UIActivity.label.PostedFrom").replace("{0}",commentAbsolutePostedTime) } else { commentAbsolutePostedTime }%>" href="<%=uicomponent.getActivityPermalink(activityID)%>" <% if(comment.postedTime != comment.updated.getTime() ){%>> $commentEditedTime <%} else {%> > ${commentPostedTime} <%}%></a>
```

Comment:
What is the purpose of this added class "editCommentTimeComment" especially that its definition does not exist.